EdFi.Buzz.Database/eng/Windows/install.ps1

```diff
@@ -69,6 +72,22 @@ function Install-Migrations {
     }
 }
 
+function Install-SampleData {
+    if ($LoadSampleData) {
+        try {
+            Push-Location -Path $distFolder
+            Write-Host "Installing sample data" -ForegroundColor Magenta
+            &db-migrate up:sample-data --silent "$DbName" --config ./migrate-database.json | Out-File -FilePath $logFile -Append
```

Comment:
This does not work unless the user has db-migrate installed globall from the command line.

Instead, I would add a yarn task in package.json - `sample-data` - and run it like our migrate function...

```json
"sample-data": "db-migrate up:sample-data --silent --config ./migrate-database.json",
```

Then, line 80 would become...

```PowerShell
&npm run sample-data | Out-File -FilePath $logFile -Append
```